guides/common/modules/ref_best-practices-for-lifecycle-environments.adoc

```diff
@@ -0,0 +1,21 @@
+[id="best-practices-for-lifecycle-environments_{context}"]
+= Best practices for lifecycle environments
+
+* Use multiple lifecycle environment paths to implement multiple sequential stages of content consumption.
+Each stage contains a defined set of content, for example in the _Production_ lifecycle environment.
+* Default use case: Fixed stages in each lifecycle environment paths, for example _Development_, _Test_, and _Production_.
+** Promote content views to lifecycle environments.
+** To make content from _Test_ available in _Production_, promote the content view version from _Test_ to _Production_.
```

Comment:
These look like procedure steps. They break up the flow. Consider moving to another module, even if it is a high-level procedure.

Reply:
I have reworded some bullet points to make it sound less like instructions. What do you think?